libraries/AP_HAL_ChibiOS/hwdef/MatekH7A3/hwdef.dat

```diff
@@ -17,210 +17,161 @@ env OPTIMIZE -Os
 FLASH_RESERVE_START_KB 32
 
 # ChibiOS system timer
-STM32_ST_USE_TIMER 12
-define CH_CFG_ST_RESOLUTION 16
+STM32_ST_USE_TIMER 5
+define CH_CFG_ST_RESOLUTION 32
 
-# USB
-PA11 OTG_HS_DM OTG1
-PA12 OTG_HS_DP OTG1
-
-PA13 JTMS-SWDIO SWD
-PA14 JTCK-SWCLK SWD
+define HAL_STORAGE_SIZE 15360
 
-# SPI1 for IMU1 (MPU6000)
-PA5 SPI1_SCK SPI1
-PA6 SPI1_MISO SPI1
-PD7 SPI1_MOSI SPI1
-PC15 IMU1_CS CS
+# use last 4 pages for flash storage
+# H7A3 has 256 pages of 8k each
+STORAGE_FLASH_PAGE 252
 
-# SPI2 for MAX7456 OSD
-PB12 MAX7456_CS CS
-PB13 SPI2_SCK SPI2
-PB14 SPI2_MISO SPI2
-PB15 SPI2_MOSI SPI2
+# use double page size for flash storage to effectively
+# give 16k pages
+define AP_FLASH_STORAGE_DOUBLE_PAGE 1
```

Comment:
I've no idea how this line is supposed to work.... there's no handling of it anywhere in the codebase.